auto update model_info [skip ci]

nvauto opened the pull request:
This PR is automatically created to update model_info.json

nvauto committed:
auto update model_info

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

ericspod committed:
Merge branch 'dev' into auto-update-model-info

KumoLiu commented:
/build

yiheng-wang-nv commented:
Hi @KumoLiu @ericspod , since the new added model requires an internal application (to host and make it public), let's block this PR merge so far, Thanks!
I will merge after getting approved.

KumoLiu commented:
Hi @yiheng-wang-nv, please also request for this one: https://github.com/Project-MONAI/model-zoo/pull/721
Thanks.

ericspod commented:
> Hi @KumoLiu @ericspod , since the new added model requires an internal application (to host and make it public), let's block this PR merge so far, Thanks! I will merge after getting approved.

Thanks @yiheng-wang-nv I'd like to get #722 through as well shortly so we can batch the three changes together I hope.

yiheng-wang-nv committed:
Merge branch 'dev' into auto-update-model-info

yiheng-wang-nv committed:
Update model info json (#725)

Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:
<!--- Put an `x` in all the boxes that apply, and remove the not
applicable items -->
- [x] Codeformat tests passed locally by running `./runtests.sh
--codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an
existing bundle.
- [ ] Please ensure the naming rules in config files meet our
requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy`
are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as
`eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing
download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config
files (such as use `/home/your_name/` for `"bundle_root"`).

---------

Signed-off-by: Yiheng Wang <vennw@nvidia.com>
Signed-off-by: Yiheng Wang <68361391+yiheng-wang-nv@users.noreply.github.com>
Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

yiheng-wang-nv committed:
Update model info (#728)

Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:
<!--- Put an `x` in all the boxes that apply, and remove the not
applicable items -->
- [x] Codeformat tests passed locally by running `./runtests.sh
--codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an
existing bundle.
- [ ] Please ensure the naming rules in config files meet our
requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy`
are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as
`eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing
download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config
files (such as use `/home/your_name/` for `"bundle_root"`).

---------

Signed-off-by: Yiheng Wang <vennw@nvidia.com>
Signed-off-by: Yiheng Wang <68361391+yiheng-wang-nv@users.noreply.github.com>
Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

yiheng-wang-nv commented:
Hi @KumoLiu @ericspod , the application to host in NGC is blocked due to NGC's changes.
Therefore, I switched to host these three new bundles into github release instead.
I will submit a related PR in MONAI to adapt the changes.

yiheng-wang-nv commented:
/build
